src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java

```diff
@@ -5,91 +5,94 @@
-    CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
-    cnw1.typeInWikiName(cnw1.getWikiName());
-    cnw1.verifySuccessIcon();
-    CreateNewWikiLogInSignUpPageObject cnwLogin = cnw1.submitToLogInSignUp();
-    cnwLogin.typeInUserName(credentials.userName);
-    cnwLogin.typeInPassword(credentials.password + "1");
-    cnwLogin.submitLogin();
-    cnwLogin.verifyInvalidPasswordValidation();
-  }
+        Assert.assertTrue(authModal.isOpened());
 
-  @Test(groups = {"CNW", "CreateNewWikiLoggedOut_003"})
-  @RelatedIssue(issueID = "SOC-2283", comment = "Test manually: Test is failing do to out of date test")
-  public void CreateNewWiki_003_blankPassword() {
-    WikiBasePageObject base = new WikiBasePageObject();
-    CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
-    cnw1.typeInWikiName(cnw1.getWikiName());
-    cnw1.verifySuccessIcon();
-    CreateNewWikiLogInSignUpPageObject cnwLogin = cnw1.submitToLogInSignUp();
-    cnwLogin.typeInUserName(credentials.userName);
-    cnwLogin.submitLogin();
-    cnwLogin.verifyBlankPasswordValidation();
-  }
+        authModal.login(credentials.userName10, credentials.password10);
+        CreateNewWikiPageObjectStep2 cnw2 = new CreateNewWikiPageObjectStep2(driver);
+        cnw2.selectCategory(CreateWikiMessages.WIKI_CATEGORY);
+        CreateNewWikiPageObjectStep3 cnw3 = cnw2.submit();
+        cnw3.selectThemeByName(CreateWikiMessages.WIKI_THEME);
+        ArticlePageObject article = cnw3.submit();
+        article.verifyWikiTitleOnCongratualtionsLightBox(wikiName);
+        article.closeNewWikiCongratulationsLightBox();
+        article.verifyWikiTitleHeader(wikiName);
+        article.verifyUserLoggedIn(credentials.userName);
+    }
 
-  @Test(groups = {"CNW", "CreateNewWikiLoggedOut_004"})
-  @RelatedIssue(issueID = "SOC-2283", comment = "Test manually: Test is failing do to out of date test")
-  public void CreateNewWiki_004_blankUserName() {
-    WikiBasePageObject base = new WikiBasePageObject();
-    CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
-    cnw1.typeInWikiName(cnw1.getWikiName());
-    cnw1.verifySuccessIcon();
-    CreateNewWikiLogInSignUpPageObject cnwLogin = cnw1.submitToLogInSignUp();
-    cnwLogin.submitLogin();
-    cnwLogin.verifyEmptyUserNameValidation();
-  }
+    @Test(groups = {"CNW", "CreateNewWikiLoggedOut_002"}, enabled = false)
+    @RelatedIssue(issueID = "SOC-2283", comment = "Test manually: Test is failing do to out of date test")
+    public void CreateNewWiki_002_wrongPassword() {
+        WikiBasePageObject base = new WikiBasePageObject();
+        CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
+        cnw1.typeInWikiName(cnw1.getWikiName());
+        cnw1.verifySuccessIcon();
+//    CreateNewWikiLogInSignUpPageObject cnwLogin = cnw1.clickNextToSignIn();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)